llvm/lib/Target/EVM/EVMStackifyCodeEmitter.cpp

```diff
@@ -357,14 +336,12 @@ void EVMStackifyCodeEmitter::createStackLayout(const Stack &TargetStack) {
             Emitter.emitDUP(static_cast<unsigned>(Depth + 1));
             return;
           }
-          if (!isRematerializable(Slot)) {
-            std::string VarName = SlotVariableName(Slot);
+          if (Slot->isRematerializable()) {
             std::string Msg =
-                ((VarName.empty() ? "slot " + stackSlotToString(Slot)
-                                  : Twine("variable ") + VarName) +
-                 " is " + std::to_string(Depth - 15) +
-                 " too deep in the stack " + stackToString(CurrentStack))
-                    .str();
+                (isa<VariableSlot>(Slot) ? "variable "
+                                         : "slot " + Slot->toString()) +
+                " is " + std::to_string(Depth - 15) +
+                " too deep in the stack " + stackToString(CurrentStack);
```

Comment:
Please, replace
```
(isa<VariableSlot>(Slot) ? "variable "
                                         : "slot " + Slot->toString())
```
with 
`(isa<VariableSlot>(Slot) ? "variable "    
                                         : "slot ") + Slot->toString()`
otherwise variable name will not be printed.